### PR TITLE
Added rules now that the repo is open

### PR DESCRIPTION
**Pull request by clayjohn**
Let me know what you think. I would like to have these posted ASAP.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,13 +6,50 @@ and later on implementing features or enhancements in Godot Engine,
...
 repository.
 
-**Note:** Work in progress! Bear with us while we discuss and define
-the exact workflow that we want to use. Please refrain from opening
-proposals on this repository just yet, unless you are among the core
-contributors and discuss it beforehand.
+#### Rules for Submitting a Proposal
+
+1. Only proposals that properly fill out the template will be considered. If
+the template is not filled out or is filled out improperly it will be closed.
```
Comment (Calinou) [inline README.md]:
```suggestion
the template is not filled out or is filled out improperly, it will be closed.
```

File: README.md [hunk 2]
```diff
@@ -6,13 +6,50 @@ and later on implementing features or enhancements in Godot Engine,
...
+
+2. All proposals must be linked to a substantive use-case. In justifying your
+proposal it is not enough to say it would be "nice" or "helpful". Use the
```
Comment (Calinou) [inline README.md]:
```suggestion
proposal, it is not enough to say it would be "nice" or "helpful". Use the
```

File: README.md [hunk 3]
```diff
@@ -6,13 +6,50 @@ and later on implementing features or enhancements in Godot Engine,
...
+template to show how Godot is not currently meeting your needs and then
+explain how your proposal will meet a particular need. 
+
+3. Proposals have similar requirements to Pull Requests (PRs) in the main repository.
+The graph in [this article](https://godotengine.org/article/will-your-contribution-be-merged-heres-how-tell)
+outlines what core developers consider when evaluating your PR/proposal.
+
+4. Other users must express interest in your proposal for it to be considered.
+Godot is community driven, if no other users are interested in your proposal,
```
Comment (Calinou) [inline README.md]:
```suggestion
Godot is community-driven, if no other users are interested in your proposal,
```

File: README.md [hunk 4]
```diff
@@ -6,13 +6,50 @@ and later on implementing features or enhancements in Godot Engine,
...
+it may be closed. It is up to you to draw interest in your proposed feature.
+Start by reaching out on the community channels (Reddit, Discord, IRC, etc.
+see the [Community Channels](http://docs.godotengine.org/en/stable/community/channels.html) doc),
+then create your proposal once you have gained some interest.
+
+5. Feel free to make a PR before a proposal. If it is a large change a core
```
Comment (Calinou) [inline README.md]:
```suggestion
5. Feel free to make a PR before a proposal. If it is a large change, a core
```

File: README.md [hunk 5]
```diff
@@ -6,13 +6,50 @@ and later on implementing features or enhancements in Godot Engine,
...
+developer may require a proposal be made and discussed before merging.
```
Comment (Calinou) [inline README.md]:
```suggestion
developer may require a proposal to be made and discussed before merging.
```

File: README.md [hunk 6]
```diff
@@ -6,13 +6,50 @@ and later on implementing features or enhancements in Godot Engine,
...
+
+6. If you or another user is capable of making a PR, include that fact in 
+the issue or in a subsequent comment so that a core contributor can
```
Comment (Calinou) [inline README.md]:
```suggestion
the issue or in a subsequent comment, so that a core contributor can 
```

File: README.md [hunk 7]
```diff
@@ -6,13 +6,50 @@ and later on implementing features or enhancements in Godot Engine,
...
+
+If your proposal was closed because the template was not filled out, then 
+fill out the template and asked the person who closed the issue to re-open it.
+
+If your proposal was closed because of lack of interest, then try to build up
+some interest on the [Community Channels](http://docs.godotengine.org/en/stable/community/channels.html)
+and then ask the person who closed the issue to re-open it.
+
+If your proposal was closed because a core contributor determined that it was
+not worth pursuing and you feel that it was wrongly closed, then  feel free
```
Comment (Calinou) [inline README.md]:
```suggestion
not worth pursuing and you feel that it was wrongly closed, then feel free
```

File: README.md (hunk 6 above)
Comment (clayjohn) [reply README.md]: A comma shouldn't be added here. You only place a comma before "so" when it is being used as a coordinating conjunction.

**Comment by clayjohn** [comment]
@Calinou Thanks for the review! I implemented all your suggested changes except one. 

As a side, is there a way for me to accept your suggested changes without added a second commit? It appeared that if I accepted them it would create another commit message.

**Comment by Calinou** [comment]
> As a side, is there a way for me to accept your suggested changes without added a second commit? It appeared that if I accepted them it would create another commit message.

GitHub's web interface doesn't have a way to amend commits, so I'm afraid that's not possible.

File: README.md [hunk 8]
```diff
@@ -6,13 +6,50 @@ and later on implementing features or enhancements in Godot Engine,
...
+developer may require a proposal to be made and discussed before merging.
+
+6. If you or another user is capable of making a PR, include that fact in 
+the issue or in a subsequent comment so that a core contributor can 
+fast-track the approval process.
+
+**What to do if your proposal is closed**
+
+If your proposal was closed because the template was not filled out, then 
+fill out the template and asked the person who closed the issue to re-open it.
```
Comment (StraToN) [inline README.md]: asked the person -> ask the person

**Comment by clayjohn** [comment]
@StraToN Fixed. :) Thank you!

**Comment by JFonS** [comment]
Looks great to me, I would merge it as is and we can fix any remaining issues or improvements in the future.

**Comment by reduz** [comment]
I don' t really think submitting a PR with the proposal is a good idea, it' s just more bureaucratic. IF the discussion is good, and core contributors agree that it can be a good idea, it can be tagged as approved and to me that should be enough?

**Pull request merged**